Assets/VRTK/Scripts/VRTK_PlayerJump.cs

```diff
@@ -0,0 +1,366 @@
+// Player Jump|Scripts|0240
+namespace VRTK
+{
+    using UnityEngine;
+    using System.IO;
+    using System;
+
+    /// <summary>
+    /// The player jump provides a jump behavior by tracking HMD world position over time, calculating velocities of a crouch and a spring motion, and adding velocity to the attached rigidbody on playerpresence.  It should be attached to the `[CameraRig]` object.
+    /// </summary>
+    /// <remarks>
+    /// This relies heavily on the status of the playerPresence, and will probably have to be updated as that changes.  In particular, playerPresence sometimes deactivates physics, which can leave this in a stuck state
+    /// the checkForJumpEnd method sends a startPhysicsFall with a velocity of 0 to prevent this issue ( cf. HeightAdjustTeleport )
+    /// 
+    /// Jump scaling for non-fixed jump factors is calculated in CalcAdjustedFinalJumpVelocity
+    /// </remarks>
+    /// <example>
+    /// VRTK/Examples/039_CameraRig_PlayerJump
```

Comment:
Look at another script for the correct sentence to use when mentioning an example.